aws/policy/application-services.yaml

```diff
@@ -30,6 +30,7 @@ Statement:
       - s3:ListAllMyBuckets
       - s3:GetBucketNotification
       - s3:PutBucketNotification
+      - s3:DeleteObjectVersion
```

Comment:
Hi @hanumantharaomvl, can you explain this s3 permission or point me to how ssm uses it? I don't really have a problem with it, but I was unable to find any ssm docs that describe how ssm uses object versioning - that might just be my inexperience with ssm though. Thanks!

Reply:
Hi @jillr , Thank you for your time reviewing the PR and responding. This PR is basically for the Plugin https://github.com/ansible/ansible/pull/49652 which requires these permissions for the integration tests to be successful and during those integration tests these s3 bucket related operations will come into picture.  Hope this clarifies, please let me know if you need more information.

Reply:
HI @jillr , can you please review and approve ?

Reply:
The change looks good, these policy PRs need to be deployed into our AWS account for testing at a time when they won't interfere with anyone else working on related things though, I usually pick a day and do them in batches. I'm currently planning to go through this queue tomorrow.

Reply:
Thank you @jillr . I have also re-validated excluding the `s3:DeleteObjectVersion`  and the integration tests were passed. Good catch. I have excluded that in the commit - https://github.com/mattclay/aws-terminator/pull/56/commits/58ef2037e9a60ef95b7313b4c070b662b5a302f8.